src/main.cpp

```diff
@@ -412,138 +534,116 @@ GbdxmArgs* readPackArgs(const po::variables_map& vm)
         errors.push_back("Missing required metadata arguments: --" + join(cliFields, ", --"));
     }
 
-    if(!errors.empty()) {
-        for(const auto& error : errors) {
-            cerr << error << endl;
+    auto categories = args->identifier->detectCategory(*args->package);
```

Comment:
* no, it wouldn't be userful: what are you reading the metadata from?
* I'll let it throw, but handle checking that all the metadata is there in GBDXM with a better message